scripts/performance/README.md

```diff
@@ -0,0 +1,81 @@
+# Performance Scripts
+
+## Overview
+This directory contains scripts used for monitoring the performance of the EdgeX services. The scripts are written in Shell.
+
+## Structure
+- `ram_cpu.sh`: Script to monitor RAM and CPU usage of specific EdgeX services.
+- `transaction_bandwidth.sh`: Script to record the number of messages and total bytes received per second.
+- `transaction_latency.sh`: Script to record the latency of each message transmission from southbound to northbound.
+
+## Prerequisites
+- Copy the `ram_cpu.sh`, `transaction_bandwidth.sh`, and `transaction_latency.sh` scripts to STM32MP157.
```

Comment:
Prior to using these scripts, there are some packages that should be installed first.  For example, ram_cpu.sh uses `pidstat`, which requires `sysstat` package. transaction_bandwidth.sh uses `mosquitto_sub` and `jq`, which requires `mosquitto-clients` and `jq` respectively.

Please add such information into prerequisite.

Reply:
updated